app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```
@@ -0,0 +1,102 @@
+@import "mixins/grid-helper";
+@import "mixins/prefixed-transform";
+
+.gem-c-cards__heading {
+  margin: 0 0 govuk-spacing(6) 0;
+}
+
+.gem-c-cards__list {
+  list-style: none;
+  padding: 0;
+  // Remove the outermost left and right margin from the internal margin of the list items.
+  margin: 0 govuk-spacing(-3);
+
+  display: grid;
+  grid-auto-flow: row;
+  // Use CSS grid to calculate the number of rows
+  grid-template-rows: auto;
+  // Use the tallest cell in the grid to set the height for all rows
+  grid-auto-rows: fractions(1);
+  grid-template-columns: fractions(1);
+
+  @include govuk-media-query($from: "tablet") {
+    grid-template-columns: fractions(2);
+  }
+
+  @include govuk-media-query($from: "desktop") {
+    // Note that browsers that don't support CSS grid display the component as one column on all
+    // breakpoints
+    grid-template-columns: fractions(3);
+
+    // For browsers that don't support CSS grid, constrain the width to 50% to improve usability
+    // especially for screen magnifier users
+    width: 50%;
+
+    // Reset the width for browsers that support CSS grid
+    @supports (display: grid) {
+      width: initial;
+    }
+  }
+}
+
+.gem-c-cards__list--two-column-desktop {
+  @include govuk-media-query($from: "tablet") {
+    grid-template-columns: fractions(2);
+  }
```

Isn't this the same as lines 22 to 24?

https://github.com/alphagov/govuk_publishing_components/blob/72fcb930d1a78c51ce33f8e1bab23a7e8c712592/app/assets/stylesheets/govuk_publishing_components/components/_cards.scss#L22-L24

If this needs to be two columns on desktop, should it be changes to `$from: "desktop"`?

```suggestion
  @include govuk-media-query($from: "desktop") {
    grid-template-columns: fractions(2);
  }
```